README.md

```diff
@@ -1,65 +1,149 @@
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/usage/options) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+### Prerequisites
 
-## Test/Deployment Actions
+In order to run this project you need:
 
-Feel free to add your own deployment actions which can run your tests and deploy to Heroku.
+- Nodejs
+- A text editor of your choice
+- A properly working pc or laptop
 
-Make sure that you do not modify the [`.github/workflows/linters.yml`](.github/workflows/linters.yml) but that you create a separe GitHub Action workflow file for that.
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/naolkidanu/game-web-app.git
+```
+
+### Install
+
+Install this project with node packages:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm jest
+```
+
+### Deployment
+
+You can deploy this project using: GitHub but if you want to deploy make sure you let me know first :3
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Naol Kidanu Merdasa**
+
+- GitHub: [Naol Kidanu Merdasa](https://github.com/naolkidanu/)
+- Twitter: [Naol Kidanu Merdasa](https://twitter.com/Naolkidanu)
+- LinkedIn: [Naol Kidanu Merdasa](https://linkedin.com/in/naol-kidanu-3b64b7227/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [x] **Design for Home page**
+- [x] **Design for Detail page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/naolkidanu/fun-zone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please, share it with your friends.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Nelson Sakwa on Behance for letting me build this awesome project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [ ] As indicated by the previous reviewer, kindly check your license link and be sure the file actually exists:wink: